api/handlers/deployment_test.go

```diff
@@ -92,6 +92,18 @@ var _ = Describe("Deployment", func() {
 			})
 		})
 
+		When("runner is not capable of rolling deploy", func() {
```

Comment:
I think you meant to change this description to be different from the one above it.